packages/contracts/src/constants.ts

```diff
@@ -45,39 +44,79 @@ export const foxEthStakingContractAddresses = [
 // Exported as a string literal for contract address discrimination purposes
 export const ETH_FOX_POOL_CONTRACT_ADDRESS = '0x470e8de2eBaef52014A47Cb5E6aF86884947F08c' as const
 
-// Exported as a string literal for contract address discrimination purposes
-export const UNISWAP_V2_ROUTER_02_CONTRACT_ADDRESS =
-  '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D' as const
+// Uniswap V2
+export const UNISWAP_V2_ROUTER_02_MAINNET = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D' as const
```

Comment:
contract naming consistency suggestion - at least include `CONTRACT` or `CONTRACT_ADDRESS` (contract is fine imo, but see both in here), on all contract address constants.

also a question, is `MAINNET` intended to suggest the address is the same across all evm networks? or is this talking about ethereum mainnet? non obvious and a bit confusing imo if we want to clean up that naming convention.

Reply:
> also a question, is MAINNET intended to suggest the address is the same across all evm networks? or is this talking about ethereum mainnet? non obvious and a bit confusing imo if we want to clean up that naming convention.

Unsure as this is unrelated to this PR and is from existing implementation. I'm tempted to leave it as-is unless necessary to change in case it makes things confusing in wherever these are used

Reply:
I've updated the address naming to be consistent throughout though :)

Reply:
> > also a question, is MAINNET intended to suggest the address is the same across all evm networks? or is this talking about ethereum mainnet? non obvious and a bit confusing imo if we want to clean up that naming convention.
> 
> Unsure as this is unrelated to this PR and is from existing implementation. I'm tempted to leave it as-is unless necessary to change in case it makes things confusing in wherever these are used

For posterity, while the name is unchanged from the existing implementation, the difference was the constants were defined inline within `unchained-client/src/evm/ethereum` in at least some cases in which `MAINNET` made sense in this context as being directly related to eth mainnet (also considering there was ropsten constants). In the global contracts package, this context is lost and mainnet become vague considering all of our chains are mainnet.